vtexdocs/RapiDoc
# Feature/magic block

#### What is the purpose of this pull request?

To render magic blocks and plain text contained in the description attributes in a OpenAPI document.

#### What problem is this solving?

The absence of body content, which includes magic blocks, in the API reference pages.

#### How should this be manually tested?

Go to [the preview page](https://deploy-preview-100--elated-hoover-5c29bf.netlify.app/docs/api-reference/catalog) and check if the magic blocks are correctly rendered.
It can be tested locally by following the step 4 described at the [RFC Developers Portal](https://www.notion.so/vtexhandbook/RFC-Developers-Portal-0e367267fbd3482b9b92e4ecd6228bd4#281058b12aa94f2c91a9f43ac03f9ed7).

#### Screenshots or example usage

[Further information and screenshots can be found here.](https://www.notion.so/vtexhandbook/Tratar-blocos-OpenAPI-com-Rapidoc-0f5a21023fbe48ce9ccc17a56d5824ed#b70ee7a657b54016bbb4aa78759ffad9)

#### Types of changes
